aws-datastore/src/androidTest/java/com/amplifyframework/datastore/ModelProviderFactory.java

```diff
@@ -1,47 +0,0 @@
-/*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-package com.amplifyframework.datastore;
-
-import com.amplifyframework.core.model.Model;
-import com.amplifyframework.core.model.ModelProvider;
-
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Set;
-
-final class ModelProviderFactory {
```

Comment:
There is a very similar thing in the _source_ code, now, called a `SimpleModelProvider`. All code now uses that, instead.